src/util/Constants.js

```diff
@@ -708,38 +708,28 @@ exports.APIErrors = {
 };
 
 /**
- * The value set for a guild's default message notifications, e.g. `ALL`. Here are the available types:
- * * ALL
- * * MENTIONS
- * @typedef {string} DefaultMessageNotifications
+ * The value set for a guild's default message notifications, e.g. `ALL_MESSAGES`. Here are the available types:
+ * * ALL_MESSAGES
+ * * ONLY_MENTIONS
+ * @typedef {string} DefaultMessageNotificationLevel
```

Comment:
Why is this renamed, when the property has the same name

Reply:
That is how all the enums are documented. The enum is named plural and the type of elements it stores have the singular name. Eg. enum: `NSFWLevels`, the type it stores: `NSFWLevel`

Reply:
👍